### PR TITLE
Bumping dependencies on neo-push client

### PR DESCRIPTION
**Pull request by tbwiss**
Bumping the dependencies for neo-push client and fix the issues that popped up.

**Additionally**
Update the webpack config file of the `graphql-toolbox` to use `path` so it's in line with the neo-push client.

### CODE REVIEW

**Review by darrellwarde — DISMISSED** [review]
LGTM!

**Comment by darrellwarde** [comment]
Also marking as a draft as not doing this for the time being.

File: examples/neo-push/client/webpack.config.js [hunk 1]
```diff
@@ -56,7 +56,9 @@ module.exports = {
         }),
     ],
     devServer: {
-        contentBase: path.join(__dirname, "dist"),
+        static: {
```
Comment (tbwiss) [inline examples/neo-push/client/webpack.config.js:59]: @tbwiss sync this toolbox webpack file

**Comment by tbwiss** [comment]
This should be good to go now.
Both the neo-push client and server are running, all issues after the dependency update are fixed. I also made some small changes to the Toolbox webpack config.
If there is no complaint I'll merge this early next week.
Pinging @darrellwarde for visibility.

**Comment by tbwiss** [comment]
One thing is odd though. If I try to execute `ts-node test-api.ts` in `/packages/graphql` I get an error that `ts-node` is not available. The same happened to the Toolbox with `webpack` on `yarn start` (which executes `webpack serve`), `webpack` was not available.

**Comment by darrellwarde** [comment]
> One thing is odd though. If I try to execute `ts-node test-api.ts` in `/packages/graphql` I get an error that `ts-node` is not available. The same happened to the Toolbox with `webpack` on `yarn start` (which executes `webpack serve`), `webpack` was not available.

Do you have `ts-node` installed globally?

And regarding `webpack`, has that been installed as a dev dependency in Toolbox and have you done a `yarn install` since pulling down the branch?

**Review by darrellwarde — APPROVED** [review]
Looks good to me as far as I can tell, let's just sort this problem with webpack in the Toolbox?

**Comment by tbwiss** [comment]
I don't have `ts-node` installed globally, and it worked before.
Yes `webpack` has been a dev dependency since the start of the toolbox package, no change there.
And I've cleaned out all the `node_modules` and ran `yarn install` several times, no luck there.

File: packages/graphql-toolbox/package.json [hunk 2]
```diff
@@ -28,9 +28,9 @@
         ]
     },
     "scripts": {
-        "build": "cross-env NODE_ENV=production webpack --config webpack.config.js",
-        "start": "cross-env NODE_ENV=development webpack serve",
-        "test-build": "cross-env NODE_ENV=test webpack --config webpack.config.js",
+        "build": "cross-env NODE_ENV=production npx webpack --config webpack.config.js",
+        "start": "cross-env NODE_ENV=development npx webpack serve",
+        "test-build": "cross-env NODE_ENV=test npx webpack --config webpack.config.js",
```
[the remark this answers is not in the document]
Comment (tbwiss) [reply packages/graphql-toolbox/package.json:33]: I agree! I'll have to check what went wrong.

[the remark this answers is not in the document]
Comment (tbwiss) [reply packages/graphql-toolbox/package.json]: That is resolved now with https://github.com/neo4j/graphql/pull/1232/commits/5ce4d49d541cbf0b1e629388d4db38cce05b6853

**Comment by tbwiss** [comment]
All should be resolved now. Merging this.

**Pull request merged**